metricbeat/module/zookeeper/_meta/kibana/7/Metricbeat-zookeeper-overview.json

```diff
@@ -0,0 +1,477 @@
+{
```

Comment:
This file belongs into the `dashboard` directory inside `7`, otherwise import does not work.